combozork.js

```diff
@@ -1,254 +0,0 @@
-    }
-}
-async function play5() {
-    let answer = await ask(opening)
-    if (answer.includes("beer")) {
-        console.log("Beer in hand you check out the scene. \nYou spot your friends sitting around a table. \nCongratulations you made it to the party.")
-        beer.take()
-
-
-    }
-    else if (answer.includes("use")) {
-        if (beer.use() === "used") {
-            status = status - 1
-            if (status < 1) {
-                console.log("you are too drunk to move. you sit down and pass out")
-                process.exit()
-            }
-
-        }
-
-    }
-
-    else if (answer.includes("truly")) {
-        truly.take()
-        while (true) {
-            console.log("A Truly is now in your inventory. Drop it immediately")
-            answer = await ask(opening)
-            if (answer.includes("truly" || answer.includes("drop") || answer.includes("down"))) {
-                truly.drop()
-                break
-            }
-            else {
-                console.log("You cannot walk round with that")
-            }
-        }
-    }
-    else if (answer.includes("exit")) {
-        console.log("you have returned to the foyer")
-        play4()
-    }
-    else if (answer.includes("pockets")) {
-        displayInventory()
-    }
-    else {
-        console.log("you cannot do that right now")
-    }
-    play5()
-}
-async function play6() {
-    let answer = await ask(opening)
-    if (answer.includes("clothes")) {
-        console.log("you move the clothes aside \nand see a small door in the \nback of the closet. \nyou see a key resting on the \nframe of the small door")
-    }
-    else if (answer.includes("take")) {
-        key.take()
-    }
-    else if (answer.includes("use")) {
-        if (key.use() === "used") {
-            console.log('You open the small door and see a cart.\nThe words "This is it" are spray painted on the wall. \nYou get in the cart and it begins to lower \nyou down a makeshift elevator shaft. \nWhen you get to the bottom of the elevator shaft \nthe cart starts rolling on rollercoaster tracks. \nThose tracks wind along until you reach another elevator shaft\nsteam hisses out of the shaft and an ominous \n"Keep out" sets in a moment of panic.\nDown the next elevator you go until you \nfinally reach the bottom. The cart stops and you get out.\nYou can hear rustling. you look around the corner \nand see a long haired man sitting in front of\na large collection of computers and some sort of lab equipment.')
-            play7()
-        }
-
-
-    }
-    else if (answer.includes("pockets")) {
-        displayInventory()
-    }
-    else {
-        console.log("you cannot do that right now")
-    }
-    play6()
-}
-
-async function play7() {
-    let answer = await ask("Who is this man?")
-    if (answer.includes("Lazlo Hollyfeld") || answer.includes("Lazlo")) {
-        console.log("Congratulations! You have won the game and soon the lottery as well")
-        process.exit()
```

Comment:
Functionally everything works, however your story is a little incomplete because if I didn't have your code up on another screen, I'd have no idea to type "lazlo"